steering/src/main.cpp

```diff
@@ -1,5 +1,6 @@
 #include "Arduino.h"
 #include "steering_io.h"
+#include "cal.h"
```

Comment:
Including this so that it gets added to compilation?

Reply:
yessir